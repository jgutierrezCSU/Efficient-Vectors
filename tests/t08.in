partitionAround
5 23
23 5 13 67 27
partitionAround
4 22.5
27 11 54 18
partitionAround
20 16.5
14	8	5	48	4	34	29	11	17	2	39	11	43	4	34	16	8	41	19	44
partitionAround
21 17
14	8	5	48	4	34	29	11	17	2	39	11	43	4	34	16	8	41	19	44	50
partitionAround
21 17
50  8   5   48  19   34  29  11  17  34   39  11  43  4   2  16  8   41  4  44  14
