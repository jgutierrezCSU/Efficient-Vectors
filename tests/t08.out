Input array A is:
23 5 13 67 27 
After partitioning of A around 23, A is:
23 5 13 67 27 
Input array A is:
27 11 54 18 
After partitioning of A around 22.5, A is:
18 11 54 27 
Input array A is:
14 8 5 48 4 34 29 11 17 2 39 11 43 4 34 16 8 41 19 44 
After partitioning of A around 16.5, A is:
14 8 5 8 4 16 4 11 11 2 39 17 43 29 34 34 48 41 19 44 
Input array A is:
14 8 5 48 4 34 29 11 17 2 39 11 43 4 34 16 8 41 19 44 50 
After partitioning of A around 17, A is:
14 8 5 8 4 16 4 11 17 2 11 39 43 29 34 34 48 41 19 44 50 
Input array A is:
50 8 5 48 19 34 29 11 17 34 39 11 43 4 2 16 8 41 4 44 14 
After partitioning of A around 17, A is:
14 8 5 4 8 16 2 11 17 4 11 39 43 34 29 34 19 41 48 44 50 
